README.md

```diff
@@ -2,94 +2,12 @@
 **Maintainer:** [@dbejanishvili](https://github.com/dbejanishvili)
 **Co-Maintainers:** [@bakurits](https://github.com/bakurits) [@Gvantsats](https://github.com/Gvantsats)
 
-Splunk integration for mattermost. Enabling users to get logs and alerts from splunk server.
-To learn more about Mattermost plugins, see [Mattermost plugin documentation](https://developers.mattermost.com/extend/plugins/).
+A Splunk integration for Mattermost which enables users to get logs and alerts from Splunk server. To learn more about installing and using this plugin, see [the plugin documentation](https://mattermost.gitbook.io/splunk-plugin/).
 
-## Table of Contents
-- [License](#license)
-- [System Admin Instructions](#for-mattermost-system-admins)
-    - [Plugin Setup](#plugin-setup)
-- [User Instructions](#for-users)
-    - [Slash Commands](#slash-commands)
-- [Developer/Contributor Instructions](#for-developers-and-contributors)
-    - [Setting Up Environment](#setting-up-environment)
-    - [Building And Deployment](#building-and-deployment)
-
-## **License**
+## License
 
 This repository is licensed under the [Apache 2.0 License](https://github.com/bakurits/mattermost-plugin-anonymous/blob/master/LICENSE).
```

Comment:
```suggestion
This repository is licensed under the [Apache 2.0 License](https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/LICENSE).
```

Reply:
Why don't link to the file in this repo?

Reply:
@hanzei, good point. Fixed.